internal/proxy/hook/authz/authz.go

```diff
@@ -194,16 +202,26 @@ func (a Authz) ServeHook(res *http.Response, err error) (*http.Response, error)
 	return a.next.ServeHook(res, nil)
 }
 
-func createResources(permissionAttributes map[string]interface{}) ([]resource.Resource, error) {
+func createResources(ctx context.Context, permissionAttributes map[string]interface{}, p ProjectService) ([]resource.Resource, error) {
 	var resources []resource.Resource
 	projects, err := getAttributesValues(permissionAttributes["project"])
 	if err != nil {
 		return nil, err
 	}
 
-	orgs, err := getAttributesValues(permissionAttributes["organization"])
-	if err != nil {
-		return nil, err
+	var organizationIds []string
+	var projectIds []string
+	for _, proj := range projects {
+		project, err := p.Get(ctx, proj)
```

Comment:
With the previous changes, here you could do
```suggestion
		project, err := a.projectService.Get(ctx, proj)
```